selfdrive/car/isotp_parallel_query.py

```diff
@@ -140,8 +142,13 @@ def get_data(self, timeout, total_timeout=60.):
       cur_time = time.monotonic()
       for tx_addr in response_timeouts:
         if cur_time - response_timeouts[tx_addr] > 0:
-          if request_counter[tx_addr] > 0 and not request_done[tx_addr]:
-            cloudlog.error(f"iso-tp query timeout after receiving response: {tx_addr}")
+          if not request_done[tx_addr]:
+            if request_counter[tx_addr] > 0:
+              cloudlog.error(f"iso-tp query timeout after receiving partial response: {tx_addr}")
+            elif tx_addr in addrs_ever_responded:
+              cloudlog.error(f"iso-tp query timeout while receiving response: {tx_addr}")
+            else:
+              cloudlog.error(f"iso-tp query timeout with no response: {tx_addr}")
```

Comment:
Previous: only logged if we successfully rx'd response for one request and still had more to send (brands which tx a tester present first for example).

Now: log each case separately: 1) above, 2) still on first request or only one request and we have received frames but not final frame, and no iso-tp errors, and 3) no valid response frames at all (ECU does not exist or a cable/hardware issue).